subprojects/docs/src/docs/release/notes.md

```diff
@@ -156,44 +168,45 @@ The following Attributes have disambiguation rules defined.
 (#): Attribute disambiguation precedence
 ```
 
-### Configurable wrapper download network timeout
+#### TODO: Extendend configuration cache support for external processes 
+[Allow buildScan.background to launch external processes with configuration cache enabled gradle#20536](https://github.com/gradle/gradle/issues/20536)
 
-It is now possible to configure the network timeout for downloading the wrapper files.
-The default value is 10000ms and can be changed in several ways:
+#### TODO: Extendend configuration cache support for internal plugin
 
-From the command line:
-```shell
-gradle wrapper --network-timeout=30000
-```
+[Allow buildScan.background to launch external processes with configuration cache enabled gradle#20536](https://github.com/gradle/gradle/issues/20536)
 
-In your build scripts or convention plugins:
-```kotlin
-tasks.wrapper {
-    networkTimeout.set(30000)
-}
-```
 
-Or in `gradle/wrapper/gradle-wrapper.properties`:
-```properties
-networkTimeout=30000
-```
+<a name="execution"></a>
+### Execution
 
-See the [user manual](userguide/gradle_wrapper.html#sec:adding_wrapper) for more information.
+#### Introduced ability to explain why a task was skipped with a message
 
-### Command-line improvements
+It is now possible to provide a reason message in conditionally disabling a task using a [`Task.onlyIf` predicate](userguide/more_about_tasks.html#sec:using_a_predicate).
+```groovy
+tasks.named("slowBenchmark") {
+    onlyIf("slow benchmarks are enabled with my.build.benchmark.slow") { 
+        providers.gradleProperty("my.build.benchmark.slow").map { it.toBoolean() }.getOrElse(false)
+    }
+}
```

Comment:
```suggestion
tasks.register("slowBenchmark") {
    def slowBenchmarksEnabled = providers.gradleProperty("my.build.benchmark.slow").map { it.toBoolean() }.orElse(false)
    onlyIf("slow benchmarks are enabled with my.build.benchmark.slow") { 
        slowBenchmarksEnabled.get()
    }
}
```

`named` requires that the `slowBenchmark` task already exists. 

Accessing the gradle property in the onlyIf is invisible to configuration caching and will break that in a non-obvious way.